### ISSUE TITLE
Screen reader skips accordion content if JavaScript is disabled

### ISSUE THREAD

**Issue by pattyde** [govau/uikit#446]
- [x] I’ve read and understood the [Contributing guidelines](./CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

# What happened

For accordions that are closed by default, screen readers will skip accordion content if JavaScript is disabled. We rely on JavaScript to toggle  the value of `aria-hidden` when the accordion is toggled.

Without JavaScript, the accordion's content is expanded by default (even if it's supposed to be closed by default and has the class `au-accordion--closed`), but `aria-hidden` is still set to `true`. 

# What I expected to happen

If the content for an accordion is visible, it should be read by screen readers.

# Reproducing

- Module name: Accordion
- Module version: 4.0.0

Steps to reproduce:

1. Visit https://uikit.service.gov.au/packages/accordion/tests/site/
2. Disable JavaScript
3. Inspect element for "Accordion closed by default"
4. The HTML for the accordion will show:

```html
<div class="au-accordion__body au-accordion--closed" id="accordion-closed" aria-hidden="true">
```
5. Because `aria-hidden="true"`, the content will be skipped by screen readers
